stubs/docutils/docutils/nodes.pyi

```diff
@@ -77,6 +80,25 @@ class Node:
         ascend: bool = False,
     ) -> Node: ...
 
+# Left out
+# - def ensure_str (deprecated)
+# - def unescape (canonical import from docutils.utils)
+
+class Text(Node, str):
```

Comment:
Moved this up so the module layout matches the source file.

This makes it easier to compare